wordpress/mod.ts

```diff
@@ -14,25 +14,28 @@ import {
 } from "npm:wp-types@3.61.0";
 
 type QueryParams = Iterable<string[]>;
-export type CollectionPath = string
+export type CollectionPath = string;
 export type WP_REST_API =
-  WP_REST_API_Attachment
+  | WP_REST_API_Attachment
   | WP_REST_API_Post
   | WP_REST_API_Term
   | WP_REST_API_User;
 
-type Transformer = (data: Partial<PageData>, raw: WP_REST_API) => Partial<PageData>;
+type Transformer = (
+  data: Partial<PageData>,
+  raw: WP_REST_API,
+) => Partial<PageData>;
 export interface Transform {
   [type: string]: Transformer;
 }
 
 export interface Options {
   baseUrl: string;
-  // auth?: string | `${string}:${string}`; // Base64 | 'user:pass'
+  auth?: string | `${string}:${string}`; // Base64 | 'user:pass'
   limit: number;
```

Comment:
Maybe `limit` could be renamed to `maxPerCollection`? It would be clear IMO.

Reply:
Agree it more clean name than limit.